### PR TITLE
Upgrade/eslint

### PR DESCRIPTION
**Pull request by butlerx**
Move all configs to the root of the repo
replace  semistandard with eslint
remove unused files such as express config and olf build coscript

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+const TITLE = 'cp-zen-platform';
```
Comment (Wardormeur) [inline index.js]: Isn't this pointless considering our arch now?

File: web/index.js [hunk 2]
```diff
@@ -1,403 +1,408 @@
...
+const eventbrite = require('../lib/eventbrite.js');
+const polls = require('../lib/polls.js');
+const orgs = require('../lib/organisations.js');
+const dojos = require('../lib/dojos.js');
+const dustI18n = require('./lib/dust-i18n.js');
+const dustLoadJs = require('./lib/dust-loadjs.js');
+const dustLoadOG = require('./lib/dust-load-open-graph.js');
+
+exports.start = () => {
+  dustI18n();
```
Comment (Wardormeur) [inline web/index.js]: no need for those ?

File: web/index.js [hunk 3]
```diff
@@ -1,403 +1,408 @@
...
-  }
-}, function (err) {
-  checkHapiPluginError('cp-zen-frontend')(err);
-});
-
-server.ext('onPreAuth', function (request, reply) {
-  var translateCookie = request.state && request.state.NG_TRANSLATE_LANG_KEY;
-  if (_.isArray(translateCookie)) {
-    translateCookie = translateCookie[0];
+  if (env === 'production' || env === 'staging') {
```
Comment (Wardormeur) [inline web/index.js]: this is useless, handled by loadB

**Comment by butlerx** [comment]
eslint ignores web/public and leaves jshint to deal with it as it always has

File: web/index.js (hunk 2 above)
Comment (butlerx) [reply web/index.js]: if you look at the files they were modified to be functions but i can revert that if need be

**Comment by butlerx** [comment]
closing in favour of #1189

**Pull request closed without merging**